src/Raven.Studio/typescript/viewmodels/database/indexes/indexErrors.ts

```diff
@@ -289,6 +319,20 @@ class indexErrors extends viewModelBase {
             this.gridController().reset();
         }
     }
+
+    clearIndexErrors() {
+        const indexesToClear = this.selectedIndexNames();
+        
+        if (indexesToClear.length > 0) {
+            const clearErrorsDialog = new clearIndexErrorsConfirm(indexesToClear, this.activeDatabase(), this.allIndexesSelected());
+            app.showBootstrapDialog(clearErrorsDialog);
+            
+            clearErrorsDialog.clearErrorsTask
+                .always(() => {                    
```

Comment:
no need to refresh if you cancel in confirmation

Reply:
done